tutorials/continuous_data/20_manipulating_raw_time_domain.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+"""
+.. _manip-raw-time-domain-tutorial:
+
+Manipulating the time domain of :class:`~mne.io.Raw` objects
+============================================================
+
+This tutorial covers cropping :class:`~mne.io.Raw` objects to restrict the time
+domain, and how to concatenate :class:`~mne.io.Raw` objects (whether selections
+from the same recording or different recordings). As always we'll start by
+importing the modules we need, and loading some example data:
+"""
+
+import os
+import mne
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
```

Comment:
All of this annoying wrangling and constant importing of `os` makes me wonder if we should have an extra function in datasets that just outright returns a well formatted path to the key files.

Reply:
which files count as "key files" besides `sample_audvis_raw.fif`?  Are you thinking something like

```Python
mne.datasets.sample.get_file('raw')  # or 'stc' or whatever
```
with a set of predefined arg values that return those key files?